### ISSUE TITLE
Missing header in tk1 hardware.h

### ISSUE THREAD

**Issue by agacek** [seL4/seL4#58]
When compiling seL4 for the TK1 with profiling turned on, I get the following error:

```
/home/ajgacek/phase3-workspace/camkes/kernel/include/plat/tk1/plat/machine/hardware.h:157:9: error: implicit declaration of function 'handleOverflowIRQ' [-Werror=implicit-function-declaration]
         handleOverflowIRQ();
```

I think that `hardware.h` file is missing

```
#include <arch/benchmark_overflowHandler.h>
```

**Comment by agacek** [comment]
Fixed by 671c353.

**Comment by heshamelmatary** [comment]
Sorry for the delay, I was on a leave
